docs/usage/quickstart.rst

```diff
@@ -3,85 +3,119 @@
+     be stored.
+
+     .. code-block:: shell
+
+       $ cd metal-k8s
+       $ mkdir -p inventory/quickstart-cluster
+       $ cd inventory/quickstart-cluster/
 
-.. code-block:: ini
+  2. Create the :file:`hosts` file, which contains a listing of all hosts.
 
-    node-01 ansible_host=10.0.0.1 ansible_user=centos
-    node-02 ansible_host=10.0.0.2 ansible_user=centos
-    node-03 ansible_host=10.0.0.3 ansible_user=centos
+     .. code-block:: ini
 
-    [kube-master]
-    node-01
-    node-02
-    node-03
+        node-01 ansible_host=10.0.0.1 ansible_user=centos
+        node-02 ansible_host=10.0.0.2 ansible_user=centos
+        node-03 ansible_host=10.0.0.3 ansible_user=centos
 
-    [etcd]
-    node-01
-    node-02
-    node-03
+        [kube-master]
+        node-01
+        node-02
+        node-03
 
-    [kube-node]
-    node-01
-    node-02
-    node-03
+        [etcd]
+        node-01
+        node-02
+        node-03
 
-    [k8s-cluster:children]
-    kube-node
-    kube-master
+        [kube-node]
+        node-01
+        node-02
+        node-03
 
-Make sure to change IP-addresses, usernames etc. according to your
-infrastructure.
+        [k8s-cluster:children]
+        kube-node
+        kube-master
 
-In a second file, called :file:`kube-node.yml` in a :file:`group_vars`
-subdirectory of our inventory, we declare how to setup storage (in the
-default configuration) on hosts in the *kube-node* group, i.e. hosts on which
-Pods will be scheduled:
+    Change the IP addresses, host names, and user names to conform to
+    your infrastructure.
 
-.. code-block:: yaml
+  3. Create a :file:`group_vars` subdirectory in the inventory directory.
 
-    metal_k8s_lvm:
-      vgs:
-        kubevg:
-          drives: ['/dev/vdb']
+   - A second file, called :file:`kube-node.yml`, in :file:`group_vars`
+     subdirectory of the inventory, declares how to set up storage (in the
+     default configuration) on hosts in the *kube-node* group; that is,
+     hosts on which Pods will be scheduled
 
-In the above, we assume every *kube-node* host has a disk available as
-:file:`/dev/vdb` which can be used to set up Kubernetes *PersistentVolumes*. For
-more information about storage, see :doc:`../architecture/storage`.
+     .. code-block:: shell
 
-Entering the MetalK8s Shell
----------------------------
-To easily install a supported version of Ansible and its dependencies, as well
-as some Kubernetes tools (:program:`kubectl` and :program:`helm`), we provide a
-:program:`make` target which installs these in a local environment. To enter this
-environment, run :command:`make shell` (this takes a couple of seconds on first
-run)::
+       $ mkdir group_vars ; cd group_vars
+
+   - In :file:`group_vars`, create a file named :file:`kube-node.yml`.
+     This file declares the default storage configuration on hosts in
+     the *kube-node* group—that is, hosts on which Pods shall be
```

Comment:
Duplication ('that is, hosts on which Pods will/shall be sheduled')

Please *render the docs* and *read the resulting page*, not just some diff.